app/modifiers/inputmaskTel.js

```diff
@@ -0,0 +1,9 @@
+import { modifier } from 'ember-modifier';
+import Inputmask from 'inputmask';
+
+export default modifier(function inputmaskTel(element) {
+  new Inputmask({
+    regex: '(\\+)\\d{6}\\d*',
```

Comment:
Shouldn't it be 10 instead of 6 ? For the size of a regular belgian phone number